Handle error during connection in sleep mode.

ngraziano opened the pull request:
If TV is in sleep mode (not totally off) when I try to connect, its response  is :
```json
{"error":[7,"not power-on"],"id":1}
```
and there is not the necessary cookie for authentication but `is_connected()` return `True` 

I get sleep mode for few minutes when I turn the TV off.

ngraziano committed:
Handle error during connection in sleep mode.

If TV is in sleep mode (not totally off) when we try to connect, its response without the necessary cookie is :

{"error":[7,"not power-on"],"id":1}

antonioparraga commented on braviarc/braviarc.py:
```diff
@@ -71,8 +71,10 @@ def connect(self, pin, clientid, nickname):
             _LOGGER.info(json.dumps(response.json(), indent=4))
-            self._cookies = response.cookies
-            return True
+            resp = json.loads(response.content.decode('utf-8'))
```
does response always have a content attribute snd content is alwais not null at this point?

antonioparraga reviewed (commented):
good fix, however please see my question

ngraziano committed:
Test connect response.

Use already decoded for log response.json()
Be more permissive (None response is accepted).

ngraziano commented:
I change to use response.json(), it was use before for log so this should be good.
I also change the test to be allow null response.

antonioparraga commented on braviarc/braviarc.py:
```diff
@@ -70,9 +70,11 @@ def connect(self, pin, clientid, nickname):
-            self._cookies = response.cookies
-            return True
+            resp = response.json()
+            _LOGGER.info(json.dumps(resp, indent=4))
```
Should be debug instead of info at this point?

ngraziano committed:
Change a log level to debug.

ngraziano committed:
Add documenation to connect.

aparraga commented:
perfect!
